### ISSUE TITLE
[Request] Add a "pause all currently playing tracks" option

### ISSUE THREAD

**Issue by weepingminotaur** [Plushtoast/playlistenchantment#4]
Currently, you can stop all playing tracks by clicking the relevant button on a macro, but it would be great to be able to pause instead, so you could easily resume playing those tracks later.

**Comment by Plushtoast** [comment]
this button already does that:

![Image](https://github.com/user-attachments/assets/0b915d61-6c7d-4948-8f46-50535a2626ed)

is this what you need or is there sth else?

**Comment by weepingminotaur** [comment]
Apologies. I assumed that was a stop button, since it uses the stop icon, not the pause icon.

**Comment by Plushtoast** [comment]
true. not very standard. but while building it I noted plain stop will probably be useless or even counterproductive. I dont remember though why I did not take the pause button.
